### DISCUSSION TITLE
IDs instead of names for community containers

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#3506] [Ideas] [answered]
I know that using the name of the project makes sense, but what if a project rebrands? Will there be two folders with the same json file but different folder/file names? Will the old project folder deprecate? Will the old forever be used... etc. 

Would it not be better to use IDs? I know this is a totally different issue, but https://github.com/nextcloud/server/issues/5488 is a good example of what happens if you use hardly change able names instead of IDs.

What about having a file, for example called index.json inside the community-containers folder which says which ID is related to which named json file (there could be an index.md created out of it with a human-readable table)? And AIO gets the container jsons by reading the IDs from the env then reading the index.json and then looking for the named json file

**Comment by szaimen** [comment, marked as answer]
I would like to keep it simple for now. We can think about how to do chabge the mapping once a project renames. Hope that is fine for you!
